requirements.txt

```diff
@@ -1,110 +1,184 @@
-django-braces==1.9.0
-django-compressor==2.2
-django-contrib-comments==1.9.0
-django-cors-headers==2.4.0
-django-crispy-forms==1.7.2
-django-debug-toolbar==1.9.1
+debugpy==1.6.4
+deepdiff==1.7.0
+defusedxml==0.7.1
+Deprecated==1.2.13
+Django==3.2.15
+django-appconf==1.0.5
+django-autocomplete-light==2.3.6
+django-braces==1.15.0
+django-compressor==4.1
+django-contrib-comments==2.2.0
+django-cors-headers==3.10.1
+django-crispy-forms==1.13.0
+django-debug-toolbar==3.2.4
 django-freshly==0.1.2
-django-haystack==2.8.1
+django-haystack==3.1.1
 django-heartbeat==2.0.2
 django-ipware==2.1.0
-django-jsonfield==1.0.1
+django-jsonfield==1.4.1
 django-modeltranslation==0.12.2
 django-nose==1.4.5
-django-oauth-toolkit==1.1.0
-django-robots==2.0
-django-security==0.9.3
+django-oauth-toolkit==2.1.0
+djangorestframework==3.12.4
+django-robots==4.0
+django-security==0.12.0
 django-test-without-migrations==0.6
 django-timedeltafield==0.7.10
 django-widget-tweaks==1.4.1
-djangorestframework==3.11.0
+dnspython==2.2.1
 docker-py==1.7.2
 dominate==2.4.0
 DoubleMetaphone==0.1
-drf-haystack==1.8.5
-drf-yasg==1.17.1
-ecdsa==0.13.3
-enum34==1.1.6
-filebrowser-safe==0.5.0
-flake8==3.5.0
+dparse==0.6.2
+drf-haystack==1.8.11
+drf-yasg==1.20.0
+ecdsa==0.18.0
+email-validator==1.3.0
+exceptiongroup==1.0.4
+filebrowser-safe==1.1.1
+flake8==6.0.0
 flex==6.14.1
 flower==0.9.3
+foresite@git+https://github.com/sblack-usu/foresite-toolkit.git@94c0c2a8e21a922caa1df6cc412d5dc1a7adb189#subdirectory=foresite-python/trunk
 funcsigs==1.0.2
-#functools32==3.2.3.post2
 future==0.16.0
+GDAL==2.4.1
+geographiclib==1.52
 geojson==1.3.2
 geopy==1.16.0
-gevent==1.4.0
-grappelli-safe==0.5.1
-gunicorn==19.6.0
+gevent==22.10.2
+grappelli-safe==1.1.1
+greenlet==2.0.1
+gunicorn==20.1.0
+hsmodels@git+https://github.com/hydroshare/hsmodels.git@3197b512d2764e2a4772596fe6c288331e53223e
 html5lib==0.9999999
+idna==3.4
 inflection==0.3.1
+iniconfig==1.1.1
+ipaddress==1.0.22
 isodate==0.5.4
+itypes==1.2.0
 Jinja2==2.10.1
 jsonpointer==1.14
 jsonschema==2.6.0
+jwcrypto==1.4.2
 keepalive==0.5
-kombu==4.6.7
+kiwisolver==1.4.4
+kombu==4.6.11
+lazy-object-proxy==1.8.0
 lxml==4.4.0
-#mapnik==0.1
 Markdown==3.0.1
-MarkupSafe==1.0
+MarkupSafe==2.0.1
 matplotlib==3.1.2
-mccabe==0.6.1
-Mezzanine==4.3.1
+mccabe==0.7.0
+Mezzanine==5.1.4
 mock==1.3.0
 nameparser==0.5.7
-netCDF4==1.5.1.2 
+netCDF4==1.5.1.2
 nose==1.3.7
-nose-timer>=0.7.4,<1.0.0
+nose-timer==0.7.6
+numpy==1.16.6
 oauthlib==3.1.0
 OWSLib==0.10.3
-paramiko==2.4.2
+packaging==21.3
+paramiko==2.12.0
 pbr==1.10.0
 pep8==1.7.0
+Pillow==9.3.0
+pip==22.3.1
+pipdeptree==2.3.3
+pluggy==1.0.0
 prettytable==0.7.2
+probableparsing==0.0.1
 probablepeople==0.5.4
+psutil==4.0.0
 psycopg2==2.8.2
-ipaddress==1.0.22
 pyasn1==0.4.4
-pycodestyle==2.3.1
+pycodestyle==2.10.0
+pycparser==2.21
 PyCRS==1.0.1
-pyflakes==1.6.0
+pydantic==1.10.2
+pydebug==1.0.3
+pyflakes==3.0.1
 pylint==1.5.5
-pyproj==1.9.5.1
+PyNaCl==1.5.0
+pyparsing==3.0.9
+pyproj==3.4.0
 pysolr==3.4.0
```

Comment:
Should we be using the latest version (3.9.0)?

Reply:
updating